pkg/model/openstackmodel/servergroup.go

```diff
@@ -58,6 +58,7 @@ func (b *ServerGroupModelBuilder) buildInstances(c *fi.ModelBuilderContext, sg *
 
 		igMeta[openstack.TagClusterName] = b.ClusterName()
 	}
+	igMeta["k8s"] = b.ClusterName()
```

Comment:
The conditional on 56 only prevents an error message in protokube.  Not sure if we should do something different there opposed to duplicating tags.

Reply:
the problem is that if I do not use this tag, the bastion IG (and its instances) is not deleted. IMO adding new tag is not much.